### PR TITLE
nvdimm: add new case to test option pmem

### PR DESCRIPTION
**Pull request by huangyum**
ID: 1826200
Signed-off-by: Yumei Huang <yuhuang@redhat.com>
Depends on https://github.com/avocado-framework/avocado-vt/pull/2522.

### CODE REVIEW

**Comment by huangyum** [comment]
(1/2) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.1.x86_64.io-github-autotest-qemu.nvdimm.nvdimm_emulate: PASS (90.37 s)
 (2/2) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.1.x86_64.io-github-autotest-qemu.nvdimm.nvdimm_emulate.test_pmem: PASS (95.20 s)

**Comment by yanan-fu** [comment]
LGTM
Acked-by: Yanan Fu <yfu@redhat.com>

By the way, please remember to update the old case name.

**Comment by yanan-fu** [comment]
@luckyh Could you help review, thanks!

File: qemu/tests/cfg/nvdimm.cfg [hunk 1]
```diff
@@ -18,11 +18,11 @@
     mem-path = ${nv_backend}
     mount_dir = "/mnt"
     pre_command = "dd if=/dev/zero of=${nv_backend} bs=1M count=1024"
-    pmem = /dev/pmem0
-    format_command = "mkfs.xfs -f ${pmem}"
+    pmem_path = /dev/pmem0
```
Comment (luckyh) [inline qemu/tests/cfg/nvdimm.cfg:21]: Suggest using another name to distinguish with the pmem option.

File: qemu/tests/cfg/nvdimm.cfg [hunk 2]
```diff
@@ -39,6 +39,10 @@
             mem-path = ${nv_backend}
             del pre_command
             del post_command
+            variants:
+                - @default:
```
Comment (luckyh) [inline qemu/tests/cfg/nvdimm.cfg:43]: should be `@pmem_default`

File: qemu/tests/cfg/nvdimm.cfg [hunk 3]
```diff
@@ -39,6 +39,10 @@
...
+                - test_pmem:
```
Comment (luckyh) [inline qemu/tests/cfg/nvdimm.cfg:44]: `pmem_on` looks better

**Comment by huangyum** [comment]
@luckyh Updated, pls help review again, thanks.

**Comment by luckyh** [comment]
Thanks all.

**Pull request merged**